### DISCUSSION TITLE
CSV export Teampass -Syspass

### DISCUSSION THREAD

**Discussion by Emily-291200** [nuxsmin/sysPass#1773] [General]
Hello, 

We are transferring from teampass to syspass. For this, I would like to export the data from Teampass to Syspass in CSV format. What should be the format of the file (column names) for this to work? Is it also possible to insert permissions? If so, how is this done? 

Thank you in advance for your help

**Comment by volcano1111** [comment]
We want to do the same. If there was a possibility to import a Teampass csv it would be very cool.

**Reply by bitking** [reply]
Try the following workaround:
1. Export teamspass passwords to Keepass format
2. Import the teamspass export to Keepass
3. Export from Keepass to XML
4. Import to Syspass

**Comment by nuxsmin** [comment]
Hello, Teampass tree management is a mess, it's stored in a single string like "Parent | Child 1 | Child 1-1 | Child 1-2", so it's quite hard to develop a migration tool.
